### DISCUSSION TITLE
アイテムシステム

### DISCUSSION THREAD

**Discussion by anminmakura** [TUSB/TheUnusualSkyBlock#94] [各種システムまとめ]
主にアイテムの仕様ついてのまとめ。

**Comment by Suyarumi** [comment]
レガシーアイテムの仕様(暫定)
攻撃すると耐久が減っていき0になると壊れる。

**Comment by Suyarumi** [comment]
無限チェスト(暫定)
チェスト内にアイテムを登録できて登録したアイテムならいくらでも収納と取り出しができるチェスト。

**Comment by Suyarumi** [comment]
防具が壊れにくくなるシステム(暫定)
これのうちどれかを採用する。(複数可)
**＊このシステムはモブの制作が進んできてから(モブデバッグをやり始めるくらい)決める。**
1:防具の耐久が減りにくくなるようにする。
2:防具の耐久を増やせるようにする。
3:耐久度を回復させる手段を増やす。
4:破損した際にそのアイテムが復元される特殊Enchantを作る。

**Comment by anminmakura** [comment]
水流を上に向けて登るとき高速で登るようなアクセサリーまたは特殊防具案(要検討)
海面ジャンプできるような速度で登れたら楽しいという発想。
TUSBって意外と水流を登る動作が多いので、そういう要素があると相性良いかもしれない。

**Comment by anminmakura** [comment]
### **TUSBメモリーについてまとめ**
最初に略の意味は The Unusual Sky Block メモリーではなく
**Temporary Use Skill Bio Memoryである。**

仕組みとしてTUSBメモリーはジョブテーブルのスキルのソースを直接参照し、呼び出す特殊アイテムである。
なので自身のLv及び職業に依存せず他の職業スキルを使用することが可能だが、TUSBメモリーによって使用回数が決まっている。
またこの呼び出しでMPは消費せず、回数のみが消費される。
例として黒魔導士のLv50スキル「プチブラック」を呼び出す杖としてネビュラロッドを予定。

また登録されているスキルのソースがあれば呼び出すことは可能であるので
どの職業でも覚えられないアウタースキルも呼び出せる。
例として13vで消えた12vの黒のアイスストームを呼び出す杖も可能。
